apps/coscrad-frontend/src/components/resources/playlists/playlist-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,6 @@
+import { ICategorizableDetailQueryResult, IPlayListViewModel } from '@coscrad/api-interfaces';
+export const PlaylistDetailFullViewPresenter =
+    // eslint-disable-next-line no-empty-pattern
+    ({}: ICategorizableDetailQueryResult<IPlayListViewModel>): JSX.Element => {
```

Comment:
Instead of `{}:ICategorizable...` try `_:ICategorizable....`, i.e. use `_` as your placeholder. Then you won't need the eslint disable line.